src/main/java/org/openimmunizationsoftware/cdsi/core/logic/EvaluateForAllowableVaccines.java

```diff
@@ -166,24 +165,24 @@ public LogicResult evaluateInternal() {
         @Override
         public void perform() {
           result = YesNo.YES;
-          log("Yes. An allowable vaccine was administered ");
-          log("Setting next step: 4.9 EvaluateGender");
+          log("Yes. The vaccine dose administered was an allowable vaccine for the target dose.");
+          log("Setting next step: 6.10 EvaluateForAllowableVaccines");
         }
       });
       setLogicOutcome(1, new LogicOutcome() {
         @Override
         public void perform() {
           result = YesNo.NO;
-          log("No.  This supporting data defined allowable vaccine was not administered.");
-          log("Setting next step: 4.9 EvaluateGender");
+          log("No. The vaccine dose administered was not an allowable vaccine for the target dose.");
+          log("Setting next step: 6.9 SatifyTargetDose");
```

Comment:
I think you mixed up sections 6.10 and 6.9. 6.10 should be SatisfyTargetDose and 6.9 is EvaluateForAllowableVaccines. The Logic looks correct since the No case should still point to 6.9 just need to change the name.